scripts/core/driver.yml

```diff
@@ -95,6 +95,14 @@ etors:
       value: "$X_MAKE_VERSION( 1, 4 )"
       desc: "version 1.4"
       version: "1.4"
+    - name: "1_5"
+      value: "$X_MAKE_VERSION( 1, 5 )"
+      desc: "version 1.5"
+      version: "1.5"  
+    - name: "1_6"
```

Comment:
please remove the 1.6

Reply:
The 1.6 should stay.  It is versioned such that it will not be included in the 1.5 build of the headers/documentation.

1.6 is the current in-development spec version and it will be used for that.